src/shared/output-mode.c

```diff
@@ -1,8 +1,32 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <stdlib.h>
+
+#include "log.h"
 #include "output-mode.h"
 #include "string-table.h"
 
+void getenv_output_mode(OutputMode *output_mode) {
```

Comment:
Name this `ret` instead of `output_mode` since it's not an input argument?

Reply:
This updates the value only when the environment variable is specified. I will add a short comment about that.
Please also see https://github.com/systemd/systemd/pull/28352#discussion_r1261379140.